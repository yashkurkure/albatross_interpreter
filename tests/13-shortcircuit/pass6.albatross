var v0 int := 1;
var v1 int := 1;
var v2 int := 1;
var v3 int := 1;
var v4 int := 1;
var v5 int := 1;
var v6 int := 1;
var v7 int := 1;
var v8 int := 1;
var v9 int := 1;
fun f24714 int () { printstring("pDu67oFzsbbK4Izz9xb"); v0 := 57&67|246+182&215*56*48-25|68&151*104|28&219*242+192|52+208&46|26|173&244-74|253; return v0;}
fun f13755 int () { printstring("7aSaup5rDqdJqd8c"); v9 := 110-224-140-182+239&158|253|44*37*250*23*248*210+105&55|173-150|118&209&178&223-161&10&185+240-165*249; return v9;}
fun f38741 int () { printstring("KWd58fTNZwZJ"); v7 := 217|36&79+252&102+220|183|131+36*216|10-30-162|200+92+90|181*160|44*208+126&92&85&181*234; return v7;}
fun f9948 int () { printstring("3iDIfawkV3c"); v5 := 0; return v5;}
fun f12633 int () { printstring("zgV"); v6 := 165-167|55-254*190|43*242-54&116+167*126-155&157&7*186|48|87|199&111-53+119; return v6;}
fun f20302 int () { printstring("vN6AnII"); v8 := 0; return v8;}
fun f52466 int () { printstring("BZ2DpwYi63cqrEU"); v6 := 0; return v6;}
fun f56859 int () { printstring("jb"); v9 := 166+64&119|117|135*186+120+252|180+224&60|246*241*153+59-237*213-164-202+245*92&8&71+184&96+20|79; return v9;}
fun f65479 int () { printstring("a5ERlBWBqDtGeWujMwG"); v2 := 235-118|169|156-90+151|117|40|169*109&116|14|101|97; return v2;}
fun f57440 int () { printstring("2WWI"); v9 := 215|141|172|151|46+230+2-46&30&141+12*87-78|242-180|25&23|144|204&178*120+247+84-237; return v9;}
if (v1||v0||v4||v4||v2||v6||v6||v5||v5||v0) { f38741() ;}
if (v8&&v2&&v5&&v5&&v1&&v8&&v4&&v5&&v1&&v0) { f52466() ;}
if (v1&&v5&&v0&&v7&&v6&&v6&&v7&&v8&&v7&&v6) { f13755() ;}
if (v3&&v6&&v1&&v7&&v2&&v2&&v9&&v3&&v2&&v5) { f38741() ;}
if (v6&&v4&&v8&&v4&&v4&&v5&&v2&&v8&&v6&&v1) { f52466() ;}
if (v2&&v7&&v5&&v8&&v5&&v1&&v3&&v5&&v6&&v0) { f13755() ;}
if (v8||v0||v2||v2||v5||v4||v7||v4||v3||v1) { f20302() ;}
if (v2&&v7&&v4&&v8&&v8&&v8&&v7&&v0&&v9&&v1) { f52466() ;}
if (v7&&v9&&v5&&v6&&v7&&v8&&v4&&v3&&v2&&v8) { f38741() ;}
if (v1||v8||v2||v6||v3||v0||v3||v0||v6||v2) { f38741() ;}
exit(0);
