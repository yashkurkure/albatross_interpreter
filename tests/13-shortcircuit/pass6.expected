KWd58fTNZwZJBZ2DpwYi63cqrEUvN6AnIIKWd58fTNZwZJ
